seqr/views/apis/variant_search_api_tests.py

```diff
@@ -209,6 +210,25 @@ def test_query_variants(self, mock_get_variants, mock_get_gene_counts):
             }
         })
 
+    # Test new search with allProjectFamilies
+    @mock.patch('seqr.views.apis.variant_search_api.get_es_variants')
+    def test_query_variants_all_project(self, mock_get_variants):
+        url = reverse(query_variants_handler, args=[SEARCH_HASH])
+        _check_login(self, url)
+
+        mock_get_variants.side_effect = _get_es_variants
+
+        response = self.client.post(url, content_type='application/json', data=json.dumps({
+            'allProjectFamilies': True, 'search': SEARCH
+        }))
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertSetEqual(set(response_json.keys()), {
+            'searchedVariants', 'savedVariantsByGuid', 'genesById', 'search', 'variantTagsByGuid', 'variantNotesByGuid',
+            'variantFunctionalDataByGuid', 'familiesByGuid', 'locusListsByGuid'})
+
+        self.assertListEqual(response_json['searchedVariants'], VARIANTS_WITH_DISCOVERY_TAGS)
```

Comment:
In general, if you use a mock its important to test that the mock was called as expected. You should have `mock_get_variants.assert_called_with` somewhere in this test, the way there is in the above test

Reply:
After learning more about how mock work, I added the parameter check with mock_get_variants.assert_called_with.